src/components/nextPrevious.tsx

```diff
@@ -127,6 +127,7 @@ const NextPrevious = ({ slug }: any) => {
       }
     }
   }
+  if (currentIndex === 0) return null
```

Comment:
This was fix for the home page that was displaying `0`
![Uploading Screen Shot 2021-01-20 at 10.51.21.png…]()